packages/core/test/preview.spec.ts

```diff
@@ -142,10 +188,18 @@ describe('Preview', () => {
       expect(networks.length).to.equal(1)
       const { networkTags, executing, skipping } = networks[0]
       expect(networkTags).to.deep.equal(['optimism'])
-      expect(executing.length).to.equal(5)
-      const [gnosisSafe, sphinxModule, create2, functionCall, call] = executing
```

Comment:
The rest of the updates in this file are just adding checks to cover the funding request I added to this test.